dataverse/file.py

```diff
@@ -19,10 +19,6 @@ def __init__(self, dataset, name, file_id=None):
 
     @classmethod
     def from_json(cls, dataset, json):
-        try:
-            name = json['dataFile']['filename']
-            file_id = json['dataFile']['id']
-        except KeyError:
-            name = json['datafile']['name']
-            file_id = json['datafile']['id']
+        name = json['datafile']['name']
+        file_id = json['datafile']['id']
```

Comment:
@dwintergruen you seem to be reverting e17d8f9 which is a fix for https://github.com/IQSS/dataverse/issues/3326 . Is this intentional?